airbyte-integrations/connectors/destination-snowflake/src/test/resources/copy_gcs_config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
fyi you should avoid committing these because they contain secrets. I've deleted the keys, will regenerate them in a little bit.

Reply:
Thanks for comment, will update it with test data.